### PR TITLE
Send and retire ecocredits

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: https://github.com/regen-network/regen-registry/issues/809
Closes: https://github.com/regen-network/regen-registry/issues/889

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles 
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-components/src/components/form/CreditRetireForm.tsx [hunk 1]
```diff
@@ -74,98 +76,116 @@ const useStyles = makeStyles((theme: Theme) => ({
...
-          <LocationCountryField />
+export const BottomCreditRetireFields: React.FC<BottomCreditRetireFieldsProps> =
+  ({ mapboxToken }) => {
+    const styles = useStyles();
+    const {
+      values: { country, stateProvince, postalCode },
+      setFieldValue,
+    } = useFormikContext<RetireFormValues>();
+
+    useEffect(() => {
```
Comment (mhagel) [inline web-components/src/components/form/CreditRetireForm.tsx:118]: Thanks for moving this here

**Review by mhagel — APPROVED** [review]
This is so cool!! Excited! 
Tested.

**Review by haveanicedavid — APPROVED** [review]
LGTM - great work

**Comment by blushi** [comment]
https://github.com/Mergifyio backport redwood hambach

**Comment by blushi** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**